packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/impactanalysis/ImpactAnalysisDiagramToolControllerTests.java

```diff
@@ -135,6 +160,55 @@ public void givenDiagramWithToolWhenAskingForImpactAnalysisOnToolThenTheReportIs
 
             int nbElementModified = JsonPath.read(result, "$.data.viewer.editingContext.representation.description.diagramImpactAnalysisReport.nbElementModified");
             assertThat(nbElementModified).isEqualTo(3);
+
+            Configuration configuration = Configuration.defaultConfiguration().mappingProvider(new JacksonMappingProvider(this.objectMapper));
+            DataTree dataTree = JsonPath.parse(result, configuration).read("$.data.viewer.editingContext.representation.description.diagramImpactAnalysisReport.impactTree", DataTree.class);
```

Comment:
Parse the data into a `DataTree` instance to ease the writing of assertions.
We usually don't do that in other Sirius Web tests, so I don't know if this is something we should avoid or if it is fine